api/src/main/java/io/strimzi/api/kafka/model/EntityOperatorJvmOptions.java

```diff
@@ -23,15 +23,15 @@
 
     private static final long serialVersionUID = 1L;
 
-    private boolean gcLoggingDisabled = false;
+    private boolean gcLoggingEnabled = true;
 
-    @Description("Disable garbage collection logging")
-    public boolean isGcLoggingDisabled() {
-        return gcLoggingDisabled;
+    @Description("If the Garbage Collection logging is enabled")
```

Comment:
This reads strangely to me. "If the Garbage Collection logging is enabled _then what?_"
```suggestion
    @Description("True to enable garbage collection logging. Default: true.")
```

Reply:
It's a getter (`isGcLoggingEnabled`) and saying "True to enable ..." sounds that you have to set `true` for enabling so it sounds like a setter which is not. My first description was describing the returning `boolean` value which says "If the GC ..."

Reply:
The purpose of the `@Descrption` is not to document the getter (you can use Javadoc for that, if you really want), but to document the property for the CRD docs. It should read appropriately for that documentation. For `PersistentVolumeClaim.isDeleteClaim()` we have

```java
@Description("Specifies if the persistent volume claim has to be deleted when the cluster is un-deployed.")
```

Something like that would be better, but I wonder if @laidan6000 agrees with me that "Specifies whether..." sounds more natural?

Reply:
I know the purpose of `@Description` and I agree that adding "Specifies wheter ... " is a good option.